zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -587,23 +598,20 @@ impl<F: Field> SignVerifyChip<F> {
             )?
         };
 
-        let pk_hash_rlc = self.assign_rlc_le(
+        let pk_hash_word = self.assign_word(config, ctx, chips, "pk_hash", pk_hash);
+        self.enable_keccak_lookup(
```

Comment:
I think it would be good to update the signature of this function to replace:
```rust
        pk_hash_lo: &AssignedCell<F, F>,
        pk_hash_hi: &AssignedCell<F, F>,
```
by
```rust
        pk_hash: &Word<AssignedCell<F, F>>,
```

Reply:
fixed [cf7e444](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1418/commits/cf7e444991feb62ad14d93754d25989abd30ef57)